paths.go

```diff
@@ -64,7 +64,7 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.electronDownloadSrc = ElectronDownloadSrc(os, arch, o.VersionElectron)
```

Comment:
I'd rather 

- you put lines 63 --> 67 in a `if o.CustomElectronPath == ""`
- and **in this function**, the `else` is directly `p.appExecutable = o.CustomElectronPath,`

Otherwise this won't work